client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -325,15 +327,22 @@ export class SeoForm extends React.Component {
 					<Notice
 						status="is-warning"
 						showDismiss={ false }
-						text={
-							isSitePrivate
-								? translate(
-										"SEO settings aren't recognized by search engines while your site is Private."
-								  )
-								: translate(
-										"SEO settings aren't recognized by search engines while your site is Hidden."
-								  )
-						}
+						text={ ( function() {
+							if ( isSitePrivate ) {
+								if ( isSiteComingSoon ) {
+									return translate(
+										"SEO settings aren't recognized by search engines while your site is Coming Soon."
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'SEO settings aren't recognized by search engines while your site is Private.' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/coming-soon/copy-updates-seo-settings)